packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -58,99 +58,106 @@ class SmoothProductCardFound extends StatelessWidget {
     );
     final ProductCompatibilityHelper helper =
         ProductCompatibilityHelper.product(matchedProduct);
-    return InkWell(
-      borderRadius: ROUNDED_BORDER_RADIUS,
-      onTap: onTap ??
-          () async {
-            await Navigator.push<void>(
-              context,
-              MaterialPageRoute<void>(
-                builder: (BuildContext context) => ProductPage(product),
-              ),
-            );
-          },
-      onLongPress: () {
-        onLongPress?.call();
-      },
-      child: Hero(
-        tag: heroTag,
-        child: Ink(
-          decoration: BoxDecoration(
-            borderRadius: ROUNDED_BORDER_RADIUS,
-            color:
-                backgroundColor ?? (isDarkMode ? Colors.black : Colors.white),
-          ),
-          child: SmoothCard(
-            elevation: elevation,
-            color: Colors.transparent,
-            padding: const EdgeInsets.all(VERY_SMALL_SPACE),
-            child: Row(
-              children: <Widget>[
-                SmoothMainProductImage(
-                  product: product,
-                  width: screenSize.width * 0.20,
-                  height: screenSize.width * 0.20,
+    return Padding(
```

Comment:
Just passing by to show you [this](https://user-images.githubusercontent.com/5972966/193933108-f02afa93-eac8-48ad-ba9b-d8472439f2cb.png) :)

Reply:
Thank you @WildOrangutan, I think you've found the most effective solution for the code review!

@M123-dev I also imagined a solution where you put temporary comments before creating the PR and you remove those comments just before merging, just during the review process then.

Still, I prefer @WildOrangutan's solution.

Reply:
You're welcome 🍻 . I think chopping down build method into multiple ones would also drastically reduce this kind of changes.

Reply:
> You're welcome 🍻 . I think chopping down build method into multiple ones would also drastically reduce this kind of changes.

That's not that easy: `Widget _build` methods are not a best practice (rather create `StatelessWidget`s or `StatefulWidget`s).
Maybe we could more often put the actual code (like `onTap`) in methods instead of inlining it.

Reply:
Pure facts @WildOrangutan

https://github.com/openfoodfacts/smooth-app/issues/2621
https://github.com/openfoodfacts/smooth-app/issues/1712